binstar_client/commands/copy.py

```diff
@@ -13,10 +13,20 @@ def main(args):
 
     spec = args.spec
     channels = bs.list_channels(spec.user)
-    if args.from_channel not in channels:
-        raise errors.UserError("Channel %s does not exist\n\tplease choose from: %s" % (args.from_channel, ', '.join(channels)))
+    __name = 'label' if (args.from_label and args.to_label) else 'channel'
```

Comment:
something internal that has a simple name, I don't want to confuse this with a parameter or something. I can add comment if you think it's not clear.